tests/contrib/botocore/test.py

```diff
@@ -201,13 +219,136 @@ def test_lambda_client(self):
         assert spans
         span = spans[0]
         self.assertEqual(len(spans), 1)
-        self.assertEqual(span.get_tag('aws.region'), 'us-east-1')
+        self.assertEqual(span.get_tag('aws.region'), 'us-west-2')
         self.assertEqual(span.get_tag('aws.operation'), 'ListFunctions')
         assert_is_measured(span)
         assert_span_http_status_code(span, 200)
         self.assertEqual(span.service, 'test-botocore-tracing.lambda')
         self.assertEqual(span.resource, 'lambda.listfunctions')
 
+    @mock_lambda
+    def test_lambda_invoke_no_context_client(self):
+        lamb = self.session.create_client('lambda', region_name='us-west-2', endpoint_url='http://localhost:4566')
+        lamb.create_function(
+            FunctionName='ironmaiden',
+            Runtime='python3.7',
+            Role='test-iam-role',
+            Handler='lambda_function.lambda_handler',
+            Code={
+                'ZipFile': get_zip_lambda(),
+            },
+            Publish=True,
+            Timeout=30,
+            MemorySize=128
+        )
+
+        Pin(service=self.TEST_SERVICE, tracer=self.tracer).onto(lamb)
+
+        lamb.invoke(
+            FunctionName='ironmaiden',
+            Payload=json.dumps({}),
+        )
+
+        spans = self.get_spans()
+        assert spans
+        span = spans[0]
+
+        self.assertEqual(len(spans), 1)
+        self.assertEqual(span.get_tag('aws.region'), 'us-west-2')
+        self.assertEqual(span.get_tag('aws.operation'), 'Invoke')
+        assert_is_measured(span)
+        assert_span_http_status_code(span, 200)
+        self.assertEqual(span.service, 'test-botocore-tracing.lambda')
+        self.assertEqual(span.resource, 'lambda.invoke')
+        context_b64 = span.get_tag('params.ClientContext')
+        context_json = base64.b64decode(context_b64.encode()).decode()
```

Comment:
Does the boto integration add this tag? It doesn't seem ideal that we're including a base64 encoded blob as a tag

Reply:
boto integration is adding the parameters as tags. And since the user is responsible for encoding the parameters then we add that blob. We could add a decoded client-context as we are already doing the work to decode. But, this is existing behavior